ufo2fontir/src/source.rs

```diff
@@ -1576,7 +1576,10 @@ mod tests {
         types::GlyphName,
     };
     use fontir::{
-        ir::{AnchorKind, GlobalMetricsInstance, GlyphOrder, NameKey, PostscriptNames},
+        ir::{
+            test_helpers::Round2, AnchorKind, GlobalMetricsInstance, GlyphOrder, NameKey,
```

Comment:
you're probably going to get yelled at for unused imports, I'd import this down in the test module..